libs/auth-api-lib/src/lib/entities/models/personal-representative-right.model.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  Column,
+  DataType,
+  CreatedAt,
+  UpdatedAt,
+  Model,
+  Table,
+  PrimaryKey,
+  ForeignKey,
+  BelongsTo,
+} from 'sequelize-typescript'
+import { ApiProperty } from '@nestjs/swagger'
+import { PersonalRepresentative } from './personal-representative.model'
+import { PersonalRepresentativeRightType } from './personal-representative-right-type.model'
+
+@Table({
+  tableName: 'personal_representative_right',
+})
+export class PersonalRepresentativeRight extends Model<PersonalRepresentativeRight> {
+  @PrimaryKey
+  @Column({
+    type: DataType.UUID,
+    primaryKey: true,
+    allowNull: false,
+    defaultValue: DataType.UUIDV4,
+  })
+  id!: string
+
+  @CreatedAt
+  @ApiProperty()
+  readonly created!: Date
+
+  @UpdatedAt
+  @ApiProperty()
+  readonly modified?: Date
+
+  @ForeignKey(() => PersonalRepresentative)
+  @Column({
+    type: DataType.UUID,
+    allowNull: false,
+  })
+  @ApiProperty()
+  personalRepresentativeId!: string
+
+  @ForeignKey(() => PersonalRepresentativeRightType)
+  @Column({
+    type: DataType.STRING,
+    allowNull: false,
+  })
+  @ApiProperty()
+  rightTypeCode!: string
+
+  @ApiProperty({
+    type: () => [PersonalRepresentative],
```

Comment:
No this is the connection table between PersonalRep and PersonalRepRightType. Each record connects to gogether one of each.

Reply:
Yes I mean you are returning `[PersonalRepresentative]` instead of just `PersonalRepresentative`, which I think is reflected as an array of PersonalRepresentatives in the OpenAPI schema.

Reply:
:/ my bad